guides/index.md

```diff
@@ -61,5 +61,7 @@ common actions and workflows of interest and why they are useful.
 
 {% include_relative _subsections/perform-garbage-collection.md %}
 
+{% include_relative _subsections/configure-agent-harvester.md %}
```

Comment:
Maybe we can reorder some of the content here to keep some better logical flow. Now we have three Guides on using the Agent but they're kind of spread out. Maybe move this one up just underneath "Using the Cryostat Agent", and then move "Using Smart Triggers" just below this new one?